### DISCUSSION TITLE
how to use tailwind filter ?? Tailwind 2.1.1

### DISCUSSION THREAD

**Discussion by fuxxingbunny** [tailwindlabs/tailwindcss#4067] [Help]
backdrop-filter backdrop-blur-lg does't work.... show nothing... 

![Screen Shot 2021-04-14 at 10 49 23](https://user-images.githubusercontent.com/80838311/114651987-31c50d80-9d0f-11eb-86de-634ac4c4e3b7.png)

**Comment by alexanderbluhm** [comment]
@fuxxingbunny Can you reproduce it on https://play.tailwindcss.com/ ?
You might need opacity, e.g. `bg-opacity-50` in order to see the blur of the background.

**Comment by simonswiss** [comment]
Hey!

You can see an implementation of `backdrop-filter backdrop-blur` classes in the navbar of the new Headless UI website:

![image](https://user-images.githubusercontent.com/485747/114825329-6b7a3f00-9e09-11eb-82ce-cfb7b93d38f6.png)

https://headlessui.dev

Notice that, as @alexanderbluhm mentioned, a `bg-opacity` utility is also applied to have the blur "show" through the navbar 👍
